api/rosetta/network.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 Alvalor S.A.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package rosetta
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+
+	"github.com/optakt/flow-dps/models/identifier"
+	"github.com/optakt/flow-dps/models/rosetta"
+)
+
+type NetworksRequest struct {
+	Metadata map[string]string `json:"metadata"`
```

Comment:
The examples that they have for the `metadata` objects are really scarce. They're either empty objects, or are `{ "Issuer": "Satoshi"}`.

Do we know that the metadata will always be `map[string]string` compatible?